docs/source/setup.py

```diff
@@ -0,0 +1,28 @@
+from setuptools import setup
+
+VERSION = "0.0.1"
+
+setup(
```

Comment:
For my understanding why do we need a setup.py in the docs/source folder aswell? Don't we already have one in the main project dir?

Reply:
Yes, I had to do this, such that read-the-docs is able to build our project.
Sphinx with its autodoc module needs read-the-docs to build our project in a virtual environment. Unfortunately is read-the-docs not able to install certain python packages with C-libraries, like `recourse` with `cplex`.
As a workaround it is possible to mock these imports, but then they have to disappear also in the requirements. So this `setup.py` is only needed for this read-the-docs build process, unfortunately I havent't found another solution yet